test/test.sh

```diff
@@ -4,24 +4,16 @@ set -eux
 
 TARGET_XCP_NG_VERSION="8.2"
 
-./build.sh "${TARGET_XCP_NG_VERSION}"
+./build.sh "$TARGET_XCP_NG_VERSION"
 
-PACKAGE=emu-manager
-REPO=xcp-emu-manager
+REPOS=xcp-emu-manager
 
-git clone git://github.com/xcp-ng/$REPO /tmp/$REPO
+for REPO in ${REPOS}; do
+    REPO_PATH=/tmp/"$REPO"
+    git clone --branch "$TARGET_XCP_NG_VERSION" git://github.com/xcp-ng-rpms/"$REPO" "$REPO_PATH"
```

Comment:
You should probably git checkout $TARGET_XCP_NG_VERSION

Reply:
git clone --branch does the checkout